.travis.yml

```
@@ -12,8 +12,18 @@ matrix:
 services:
   - docker
 
-script:
-  - make test-coverage
+jobs:
+  include:
+    - name: 'Unit Tests'
+      script:
+        - make test-coverage
+    - name: 'Generated code'
+      script:
+        - GO111MODULE=on go mod tidy
+        - GO111MODULE=on go mod vendor
+        - make no-changes-in-commit
+        - (cd ./api && make)
```

I'd make it explicit:
```suggestion
        - make -C api proto
```

`make` without target is tricky, and can vary without noticing.

---

Thanks, change applied and squashed